### DISCUSSION TITLE
Using Attachment field as a card image

### DISCUSSION THREAD

**Discussion by ceobaharan** [Budibase/budibase#9806] [Help] [answered]
HI
How to use attachment field as a card component image?

**Comment by andz-bb** [comment, marked as answer]
hey @ceobaharan, your attachment field has an array of objects, one for each attachment in the field. These objects will have a URL string that you can use for the Image URL parameter in the Cards component. If you will only have one image per row in the attachment column you can use the binding for your attachment field with `.0` to select the first attachment object in the array and then `.url` to use the url from the attachment object. 

I made one here and used `{{ New Cards block.Players.Photo.0.url }}` for my binding where Players is the table and Photo is the attachment field.
<img width="761" alt="cards from attachments" src="https://user-images.githubusercontent.com/110921612/221605641-80139e52-7890-4924-a0be-cd9a4afa4732.png">

Hope this helps!

**Reply by ceobaharan** [reply]
Best Regards
